staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -263,6 +293,10 @@ type Config struct {
 	// APIServerID is the ID of this API server
 	APIServerID string
 
+	// APIServerIDConfig is the rest config used to manage identity leases
+	// If unspecified, the in-cluster config is used
+	APIServerIDConfig *restclient.Config
```

Comment:
If it's nil, can we assign this in a `Complete` so it has a value when we use it?